apps/desktop/src/app/tools/send-v2/sends.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, ChangeDetectionStrategy } from "@angular/core";
+
+@Component({
+  selector: "app-sends-v2",
+  imports: [],
+  template: "<p>Sends V2 Component</p>",
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SendsComponent {}
```

Comment:
We actually call these `send` in other places.

```suggestion
export class SendComponent {}
```

Please also rename the file to `send.component.ts`.